sympy/concrete/summations.py

```diff
@@ -96,63 +96,149 @@ class Sum(AddWithLimits, ExprWithIntLimits):
     Examples
     ========
 
+    >>> from sympy import *
+    >>> import math
+    >>> import sympy as sp
+    >>> C100 = sp.Sum(n, (n,1,100)) # Count 1 + 2 + 3 .. + 100 - Evaluating Sums
+    >>> Eq(C100, C100.doit())
+
+    .. math::
+
+        \displaystyle \sum_{n=1}^{100} n = 5050
```

Comment:
The doctests need to show what the actual output would be in a terminal.

Reply:
Just to clarify Oscar's point, our tests check that the documentation examples (doctest) output exactly what they should. So this PR won't be able to be merged unless you change the outputs here. If you also want to show LaTeX math, I would suggest adding some text before each example, like

```
    .. math::
   
       \displaystyle \sum_{n=1}^{100} n = 5050

    >>> from sympy import *
    >>> import math
    >>> import sympy as sp
    >>> C100 = sp.Sum(n, (n,1,100)) # Count 1 + 2 + 3 .. + 100 - Evaluating Sums
    >>> Eq(C100, C100.doit())
    Eq(Sum(n, (n, 1, 100)), 5050)
```